python/lsst/sims/movingObjects/orbits.py

```diff
@@ -203,76 +206,88 @@ def assignSed(self, orbits, randomSeed=None):
         return sedvals
 
     def readOrbits(self, orbitfile, delim=None, skiprows=None):
-        """Read orbits from a file, generating a pandas dataframe containing columns matching
-        dataCols, for the appropriate orbital parameter format (currently accepts COM or KEP formats).
+        """Read orbits from a file, generating a pandas dataframe containing columns matching dataCols,
+        for the appropriate orbital parameter format (currently accepts COM, KEP or CAR/CART formats).
```

Comment:
The 'CAR' keyword isn't something that I found documented anywhere (presumably it's in the DES data format standard, but the doc on that is very hard to find .. I know I saw it at some point, but have been unable to find it ever since), so I just started using "CART" for "Cartesian" instead of keeping to the 3-characters used for COMetary and KEPlerian. This has now propagated to sbpy, at least in dictionary keys ... 
https://github.com/NASA-Planetary-Science/sbpy/search?q=CART&unscoped_q=CART

So I guess I'd leave it and support both? It does mean that 'CART' may propagate further, not sure how bad that would be.

Reply:
Looks like the standard doc (http://adams.dm.unipi.it/orbfit/PS_DES_v2.02.pdf -- is that it) specifies "CAR" (see page 7); may be best to stop the propagation while it didn't go too far (and ping @mommermi about deprecating it in `sbpy`). Reducing complexity is good :).

Reply:
Interesting, I didn't even know that 'CAR' existed and I took 'CART' from Lynne's example notebook. I'm perfectly fine with switching to 'CAR'...

Reply:
Another data point is that command-line OpenOrb doesn't understand `CART`.

Reply:
Ok, I'm convinced and I'll start removing CART from here.